src/preact-redux.d.ts

```diff
@@ -20,16 +20,16 @@ export interface DispatchProp<S> {
 }
 
 interface AdvancedComponentDecorator<TProps, TOwnProps> {
-    (component: AnyComponent<TProps, {}>): ComponentConstructor<TOwnProps, {}>;
+    (component: ComponentFactory<TProps>): ComponentConstructor<TOwnProps, {}>;
```

Comment:
Would you be able to shed some light on what issue this solves?

Reply:
AnyComponent is for Component instances and ComponentFactory for factories. At least it was like so, previously.